docs/syntax_and_semantics/select.md

```diff
@@ -0,0 +1,52 @@
+# select
+
+The `select` expression chooses from a set of blocking operations and proceeds with the branch that becomes available first.
```

Comment:
Available first is not 100% accurate in MT or even if both are already available. I'm not sure how to frame the difference.

Reply:
If any actions are directly executable, `select_impl` immediately returns the result of the first action with delivery state `delivered` or `closed`. So IIUC "available first" should be good for that scenario.

With multithreading everything is a bit more complicated, of course. But even there the action that first manages to activate the select context is chosen.

Maybe we just need a different word for "available"? 
```suggestion
The `select` expression chooses from a set of blocking operations and proceeds with the branch that activates first.
```